content/blog/2022/10/envvar-policy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Enforcing compliance in your environment variables
+authors:
+- José Guilherme Vanz
+date: 2022-11-01
+---
+
+We're glad to present to Kubewarden users the new [environment-variable-policy](https://github.com/kubewarden/environment-variable-policy) policy.
+It's able to inspect containers, init containers, and ephemeral containers, and restricts
+their usage of environment variables by checking their names and values defined under the
+containers' [`env[*]`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core).
+As always, the policy can be found in [Github](https://github.com/kubewarden/environment-variable-policy)
+and all the  artifacts, including the BOM files, are signed with Sigstore
+
+### What is so useful about the new environment-variable policy?
+
+The recent added policy https://github.com/kubewarden/env-variable-secrets-scanner-policy allow users
+to check if there are some leaked sensitive data in the environment variables. However, until the
+environment-variable-policy there were no way to check other environment variables aspects.
+This is fixed now. In this new policy users can validate which variables name and values their
+resources should have. For that, the policy gives 4 rule operator to control the
+environment variable defined in the containers. They are:
+
+- `anyIn`: Works as a deny list of your usage of volumes. It checks if any of the `environmentVariables` are in the Pod/Workload resource:
+    ```yaml
+settings:
+  rules:
+    - reject: anyIn
+      environmentVariables:
+        - name: "envvar1"
+        - name: "envvar2"
+    ```
```

Comment:
I was skeptical about having unordered lists entries with nested code blocks. I tried to render the document locally and it looks bad/broken:

![image](https://user-images.githubusercontent.com/22728/197477832-aaa53305-e21c-4efc-9b6b-452a75b150b2.png)

I propose the following approach:

* Turn each bullet point into a header 2
* Inside of each section provide:
   * Explanation of what the rule (`anyIn`, `anyNotIn`, ... ) does. This is currently written after the `:`
   * Code block as
   * An example of that specific setting

Reply:
It is broken because there is a blank space before ` ```yaml`. If this space is removed the rendering is correct (see image attached) However, I would also be ok with converting each bullet point into a header 2 as flavio suggested.
![Screenshot from 2022-10-24 11-55-09](https://user-images.githubusercontent.com/6781025/197501181-df669703-1dee-4e07-98f9-379e28959173.png)